### ISSUE TITLE
Tests: ModuleNotFoundError: No module named 'babelfont.variablefont'

### ISSUE THREAD

**Issue by ctrlcctrlv** [simoncozens/fontFeatures#16]
```
[fred@laptop fontFeatures]$ python3 -m unittest tests/test_valuerecord.py 
E
======================================================================
ERROR: test_valuerecord (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_valuerecord
Traceback (most recent call last):
  File "/usr/lib/python3.9/unittest/loader.py", line 154, in loadTestsFromName
    module = __import__(module_name)
  File "/home/fred/Workspace/fontFeatures/tests/test_valuerecord.py", line 2, in <module>
    from babelfont.variablefont import VariableFont
ModuleNotFoundError: No module named 'babelfont.variablefont'


----------------------------------------------------------------------
Ran 1 test in 0.000s

FAILED (errors=1)
```

This seemingly is not a problem with my machine, as the `babelfont` module really doesn't contain a `variablefont`. But you know, given it's your package :-)

**Comment by simoncozens** [comment]
Huh, I didn't do a `git push` - but the released version (2.0.0) should have it. Try a `pip install -U babelfont`.

**Comment by ctrlcctrlv** [comment]
Thanks, the test now passes.
